builtin/logical/pki/path_config_ca.go

```diff
@@ -105,6 +105,70 @@ accessible by the existing signing paths (/root/sign-intermediate,
 /root/sign-self-issued, /sign-verbatim, /sign/:role, and /issue/:role).
 `
 
+func pathConfigKeys(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "config/keys",
+		Fields: map[string]*framework.FieldSchema{
+			"default": {
+				Type:        framework.TypeString,
+				Description: `Reference (name or identifier) of the default key.`,
+			},
+		},
+
+		Callbacks: map[logical.Operation]framework.OperationFunc{
+			logical.UpdateOperation: b.pathKeyDefaultWrite,
```

Comment:
I believe since @cipherboy already merged the fixup Jira issue for the forwarding we need to add the bits to the UpdateOperation here to forward to the main cluster. This will also need to be updated to the non-deprecated format. Sorry :(

Reply:
Updated to the non-deprecated format :), I'm pretty sure the forwarding was default for update operations, but no reason not to be explicit about it